src/index.html

```diff
@@ -4,9 +4,340 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__icon">
+        <img src="images/logo.png" alt="icon">
+      </a>
+      <nav>
+        <ul class="header__nav">
+          <a href="#" class="header__link header__link--is-active is-active">
+            Apple
+          </a>
+          <a href="#" class="header__link">
+            Samsung
+          </a>
+          <a href="#" class="header__link">
+            Смартфоны и телефоны
+          </a>
+          <a
+            href="#"
+            data-qa="nav-hover"
+            class="header__link"
+          >
+            Ноутбуки и компьютеры
+          </a>
+          <a href="#" class="header__link">
+            Гаджеты
+          </a>
+          <a href="#" class="header__link">
+            Планшеты
+          </a>
+          <a href="#" class="header__link">
+            Фото
+          </a>
+          <a href="#" class="header__link">
+            Видео
+          </a>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="foto"
+            class="card__foto"
+          >
+          <a href="#" class="card__item-model">
+            Моноблок APPLE A1419 iMac<br>27" Retina 5K (MNED2UA/A)
+          </a>
+          <p class="card__item-kod">
+            Код товара: 195434
+          </p>
+          <div class="card__feedback">
+            <div class="card__stars card__stars--amount-stars--4">
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+              <div class="card__star"></div>
+            </div>
```

Comment:
`stars` it's a separate block.

Add changes to styles according to changes in html. And move styles for `stars` block in a separate `stars.scss` file.
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```